torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2054,6 +2054,30 @@ def sample_inputs_std_var(op_info, device, dtype, requires_grad, **kwargs):
         SampleInput(tensor_1d, kwargs=dict(dim=0, unbiased=False, keepdim=False)),
     ]
 
+def sample_inputs_cov(op_info, device, dtype, requires_grad):
+    inp_dims = [ (S,), (1, S), (S, 2), (S, S)]  # (S, 1) produces nan outputs
```

Comment:
Does NumPy also return nan for (S, 1). Is this the expected behavior?

Reply:
@heitorschueroff Yes, without any other params passed, just (S,1) returns nan. Here's an example:

```
>>> a = np.random.randn(5,1)*10
>>> a
array([[ -0.55035937],
       [-10.05814517],
       [ -5.84499751],
       [  7.80409234],
       [  5.79857037]])
>>> np.cov(a)
<stdin>:1: RuntimeWarning: Degrees of freedom <= 0 for slice
/home/rohan/codes/Tensorplay/env/lib/python3.8/site-packages/numpy/lib/function_base.py:2493: RuntimeWarning: divide by zero encountered in true_divide
  c *= np.true_divide(1, fact)
/home/rohan/codes/Tensorplay/env/lib/python3.8/site-packages/numpy/lib/function_base.py:2493: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
array([[nan, nan, nan, nan, nan],
       [nan, nan, nan, nan, nan],
       [nan, nan, nan, nan, nan],
       [nan, nan, nan, nan, nan],
       [nan, nan, nan, nan, nan]])
```